config/locales/en.yml

```diff
@@ -814,7 +814,7 @@ en:
       legal_aid: Legal Aid
       agency: Agency
       print_this_page: Print this page
-      check_eligibility_for_legal_aid: Check Eligibility For Legal Aid
+      check_eligibility_for_legal_aid: Check if your client qualifies for legal aid
```

Comment:
I'd also update or remove en.cookies.show.service_name

Reply:
Looks done in d3ecab2